src/components/BalanceView.js

```diff
@@ -28,52 +29,26 @@ import Text from 'components/modern/Text';
 import { useFiatCurrency } from 'selectors';
 
 // Utils
-import { getCurrencySymbol, formatFiatValue } from 'utils/common';
-import { appFont, fontStyles } from 'utils/variables';
+import { formatFiatValue } from 'utils/format';
 
 // Types
-import type { ViewStyleProp, TextStyleProp } from 'utils/types/react-native';
-
+import type { TextStyleProp } from 'utils/types/react-native';
 
 type Props = {
-  balance: number,
-  style?: ViewStyleProp,
-  currencyTextStyle?: TextStyleProp,
-  balanceTextStyle?: TextStyleProp,
+  balance: BigNumber | number,
```

Comment:
This is intentional, as I wanted to apply more strict typing to the new code, so that all token values are BigNumber-only, and all momentary values is BigNumber | number only (here number is allowed because fiat value is usually calculated using redux rates which are themselves number)